ailab/benchmark/db_utils.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
```

Comment:
None of the method in this file is used.

Reply:
Removed the entire file from this commit.